apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/MonitoredResourceUtils.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Yes, you're right.
It's actually in GAX ([MonitoredResource.cs](https://github.com/googleapis/gax-dotnet/blob/master/src/Google.Api.CommonProtos/Api/MonitoredResource.cs)), which means this common code can go there.
Perhaps leave this as it is for now, and file an issue to move this code into GAX, which I'll do soon.